src/tests/ftest/posix/simul.py

```diff
@@ -79,13 +81,20 @@ def run_simul(self, include=None, exclude=None):
         If include value is set, exclude value is ignored and vice versa.
 
         """
-        simul_openmpi = "/usr/lib64/openmpi3/bin/simul"
-        simul_mpich = "/usr/lib64/mpich/bin/simul"
-        simul_dict = {"openmpi": simul_openmpi,
-                      "mpich": simul_mpich
-                      }
+        # Assuming all vms use the same OS
+        host_os = distro.linux_distribution()[0].lower()
+        dfuse_hosts = self.agent_managers[0].hosts
+
+        if "centos" in host_os:
+            simul_dict = {"openmpi": "/usr/lib64/openmpi3/bin/simul",
+                          }
+        elif "suse" in host_os:
+            simul_dict = {"openmpi": "/usr/lib64/mpi/gcc/openmpi3/bin/simul",
+                          }
+        else:
+            print(host_os)
```

Comment:
```suggestion
            self.log.error(host_os)
```
We usually use self.log for print.

Reply:
Fixed